tests/test_webapp.py

```diff
@@ -264,7 +308,6 @@ def test_langpack_invalid_languages_target_wrong_version_value(self):
         })
         self.analyze()
         self.assert_failed(with_errors=True)
-        self.assert_got_errid(("spec", "iterate", "bad_value", ))
```

Comment:
`self.assert_got_errid(("spec", "iterate", "value_pattern_fail", ))`

(Found by using `pprint.pprint(self.err.errors)` on this test)